cfgov/v1/atomic_elements/molecules.py

```diff
@@ -257,6 +257,7 @@ class ContactPhone(blocks.StructBlock):
     phones = blocks.ListBlock(
         blocks.StructBlock([
             ('number', blocks.CharBlock(max_length=15)),
+            ('extension', blocks.CharBlock(max_length=4, required=False)),
             ('vanity', blocks.CharBlock(max_length=15, required=False)),
```

Comment:
While we're touching this file, could we add some `help_text` to better indicate how to use this field?

Reply:
Any thoughts on what that help text should include?